src/service/performance-impl.js

```diff
@@ -536,9 +531,8 @@ export class Performance {
         // and we just need to tick `pc`. (it will give us the relative
         // time since the viewer initialized the timer)
         this.tick('pc');
-        // We don't have the actual csi timer's clock start time,
-        // so we just have to use `docVisibleTime`.
-        this.prerenderComplete_(this.win.Date.now() - docVisibleTime);
+        // TODO before merge: what should be the actual value here? performance.now() or performance.now() - docVisibleTime?
```

Comment:
I think it should be the latter. The code at HEAD passes `this.win.Date.now() - docVisibleTime`, which is the delta between two absolute dates. It's equivalent to the delta between two relative timestamps in the performance clock, so `this.win.performance.now() - docVisibleTime` in this PR.

Reply:
I tend to agree with you! The tests were getting away with counting `docVisibleTime` as 0, since they were never triggering that signal. I'll either update a test or add a new one and then submit this.